CHANGES

```diff
@@ -7,6 +7,13 @@ Features
 - The annotation text input is focused automatically when
   new annotations are created.
 
+Bug fixes
+---------
+
+- Fixed error when rendering invalid LaTeX markup. As a side effect,
+  the fallback to MathJax rendering for markup that is not supported
+  by KaTeX has been removed for the time being.
```

Comment:
Can you include the PR number in this? One way to do this consistently is to add the CHANGES entry in a commit pushed after you first open the PR.